src/Stryker.Core/Stryker.Core/Stryker.Core.csproj

```diff
@@ -72,4 +72,9 @@
   <ItemGroup>
     <ProjectReference Include="..\..\Stryker.DataCollector\Stryker.DataCollector\Stryker.DataCollector.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Reporters\TestStatisticsReporter\" />
```

Comment:
This shouldn't be neccesary on dotnet core, all source files are automatically included in the project

Reply:
it was added by VStudio, does not know why...

Reply:
removed